### ISSUE TITLE
Kopernicus Failure with Realism Overhaul; KSP 1.8.1

### ISSUE THREAD

**Issue by tj-murphy** [Kopernicus/Kopernicus#474]
Hello,

Yesterday I attempted to install Realism Overhaul for 1.8.1. I find that when I reach the main menu, it runs at about 3 frames per second, and if I try to load a game, the planet animation also runs around 3 frames per second, and the game doesn't load (at least, within any reasonable timeframe).

When I reach the main menu, I get a message box indicating Kopernicus was unable to load the custom planetary system due to an exception in the loading process.

I also see that, approximately every 0.14 seconds, I have errors in KSP.log which are of the form:
```
[EXC 09:31:31.590] NullReferenceException
	Kopernicus.RuntimeUtility.RuntimeUtility.AtmosphereLightPatch (CelestialBody body) (at <516b7fac868b4b6bb7981b26b6f52c1d>:0)
	Kopernicus.RuntimeUtility.RuntimeUtility.LateUpdate () (at <516b7fac868b4b6bb7981b26b6f52c1d>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object) 
```

These error messages seem to start at roughly the time that the main menu shows up.

Please note that I have fully uninstalled KSP and re-attempted to install the mods from scratch. If I remove Kopernicus, the game launches great (though of course I have Kerbin instead of Earth).

This is a Linux install through CKAN. I have verified that I am running 1.8.1-23.

I've attached all the logging files that the original main-menu load failure message asked for. More than happy to cooperate with any debugging requests :)

Thanks!

[kopernicus_load_fail.zip](https://github.com/Kopernicus/Kopernicus/files/5894394/kopernicus_load_fail.zip)

**Comment by R-T-B** [comment]
It seems the 1.8.3 release I posted has some bugs. I will try to fix this today, thanks for the report.

You can revert to 1.8.1-1 (not a typo) in the meantime.

**Comment by R-T-B** [comment]
This should be fixed in next release being pushed to CKAN now.  Please confirm.

**Comment by tj-murphy** [comment]
I'm not seeing 1.8.1-24 showing up in CKAN, even after clicking the Refresh button to update the repositories. CKAN continues to show 1.8.1-23.

I manually went to the Releases page on this github, found 1.8.1-24 (Not 1.9.1-24!) and pulled out the Kopernicus folder. Then I deleted the Kopernicus I already had and placed that new version in. Therefore note that I did not copy the ModularFlightIntegrator and ModuleManager from the new version, only the Kopernicus folder.

That seems to be working perfectly now. Game runs smoothly, and I'm in Cape Canaveral.

Thanks for the speedy fix!
